pom.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>uk.gov.companieshouse</groupId>
+    <artifactId>company-accounts.api.ch.gov.uk</artifactId>
+    <version>unversioned</version>
+    <packaging>jar</packaging>
+    <name>company-accounts-api</name>
+    <description>Company accounts API to handle company accounts</description>
```

Comment:
Maybe say "to handle company accounts data" (or something along those lines - makes it a bit clearer)

Reply:
Done